packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1847,6 +1848,9 @@ class EditableText extends StatefulWidget {
   /// {@macro flutter.widgets.magnifier.TextMagnifierConfiguration.details}
   final TextMagnifierConfiguration magnifierConfiguration;
 
+  /// Determine whether [onTapOutside] can be called when is not focused.
```

Comment:
Consider adding a mention of this new parameter in the documentation for `onTapOutside`.